### PR TITLE
Corrected problem in \author when I have more than two lines.

### PR DESCRIPTION
**Pull request by tjiagoM**
In my report I have to add several information in `\author`. Something like this:

`\author{My Full Name \\ Supervisor: Full Name 2 \\ Co-Supervisor: Full Name 3 \\ Second Adviser: Full Name 4}`

And this was the result (bad paragraph spacing on last line):

image

This pull request is the solution I suggest, which will turn the previous case into this one:
image

If I try with one or two lines the visual result is the same, I think.

### CODE REVIEW

File: thesis.tex [hunk 1]
```diff
@@ -37,7 +37,7 @@
 spanning two lines}
 
 %% The full name of the author (e.g.: James Smith):
-\author{My Full Name}
+\author{My Full Name \\ Supervisor: Supervisor's Name}
```
Comment (urbas) [inline thesis.tex]: Let's keep the author as it is. Could you remove the supervisor from here and from `02ThesisInfo.tex` too?

Comment (urbas) [inline thesis.tex]: @tjiagoM: bump

Comment (tjiagoM) [inline thesis.tex]: Bump? I don't get it, sorry.

Comment (urbas) [inline thesis.tex]: I "bumped" the comment to bring it to your attention.

Re "supervisor information": The default setting of `thesis.tex` is to build the thesis (not a tech report) which does not need the info about the supervisor. I think it's therefore okay if users have to manually add the supervisor.

I'm also not sure if the supervisor is strictly needed in the tech report. We should probably check somewhere. If it is needed, then we could add another field just like `\author`, say `\supervisor`, which would then only be used in the tech report.

Comment (tjiagoM) [inline thesis.tex]: Oh, I thought the supervisor information was mandatory. Indeed I searched a bit and couldn't find any information about supervisors information. My bad, I will remove that lines, then.

**Comment by urbas** [comment]
Nice! :+1: 

If you scroll to page 24 of these two PDFs, you'll spot the difference:

- before this PR: https://75-1710676-gh.circle-artifacts.com/0/cam-thesis/build/pdfs/clean.pdf

- after this PR: https://79-1710676-gh.circle-artifacts.com/0/cam-thesis/build/pdfs/clean.pdf

Looks like it works!

**Pull request merged**